experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -57,6 +57,15 @@
       - name: AWS authentication
         include_tasks: "/utils/cloud/aws/aws_configure.yml"
         when: "cloud_platform == 'AWS'"
+        
+      ## RECORD EVENT FOR PRE-CHAOS CHECK
+      - include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
```

Comment:
Can you please add this task just after checking the running status of AUT and auxiliary apps?

Reply:
@rajdas98 Please don't resolve the conversation without solving it.

Reply:
My bad forget to push. Made the changes

Reply:
Can you check now?